ecosystem/sep-0024.md

```diff
@@ -152,11 +154,9 @@ Name | Type | Description
 `wallet_url` | string | (optional) Anchor should link to this when notifying the user that the transaction has completed.
 `lang` | string | (optional) Defaults to `en`. Language code specified using [ISO 639-1](https://en.wikipedia.org/wiki/ISO_639-1). `error` fields in the response should be in this language.
 
-Example:
+Additionally, any [SEP-9](SEP-0009.md) parameters may be passed as well to make the onboarding experience simpler.
 
-```
-GET https://api.example.com/deposit?asset_code=ETH&account=GACW7NONV43MZIFHCOKCQJAKSJSISSICFVUJ2C6EZIW5773OU3HD64VI
-```
+When uploading data for fields specificed in [SEP-9](./sep-0009.md), `binary` type fields (typically files) should be submitted **after all other fields**. The reason for this is that some web servers require `binary` fields at the end so that they know when they can begin processing the request as a stream.
```

Comment:
Would it be feasible to add a real example here, like we had before? I feel like that has value.

Reply:
I added the parameters to the request example at the top, does that help?

Reply:
actually i don't like it there, going to update it